### DISCUSSION TITLE
핵불닭만큼 급합니다! 꼭 답변 부탁드립니다.

### DISCUSSION THREAD

**Discussion by answerpleasebro** [naverz/zepeto-studio-kor#8] [5. Q&A]
저희 게임에서는 현재 귀사가 제공하지 않는 모션을 사용하여야 합니다.
모션을 만들기 위해서는 기존 제페토 캐릭터의 마야(3d) 파일이 필요할것같습니다.
이부분은 제공해주시지 않으시나요?
문의드립니다.
제공해주시지 않으신다면 어떤 방법으로 모션을 제작해야 하나요?
모션제작에대한 가이드가 있으시다면 가이드도 함께 답변 부탁드립니다.

**Comment by x4cf** [comment]
https://studio.zepeto.me/guides/creating-your-item-3d-modelers-1
에 들어가셔서
creatorBaseSet_zepeto.zip 파일 (다운로드) - 2021.08 updated 파일을 다운로드 받으신다음 작업하시면됩니다
